config/initializers/omniauth_idcat_mobil.rb

```diff
@@ -0,0 +1,9 @@
+Devise.setup do |config|
+  config.omniauth :idcat_mobil,
+                  ENV["IDCAT_MOBIL_CLIENT_ID"],
+                  ENV["IDCAT_MOBIL_CLIENT_SECRET"],
+                  ENV["IDCAT_MOBIL_SITE_URL"],
+                  scope: :autenticacio_usuari
```

Comment:
in english?

Reply:
no, IdCat mòbil requires this parameter to be in catalan.

Reply:
ok